### DISCUSSION TITLE
Link the documentation instead of pypi in the about box.

### DISCUSSION THREAD

**Discussion by sairam4123** [nextcord/nextcord#131] [General]
Currently the in the about box, the pypi download link is shown, maybe replace it with documentation link to make documentations easily accessible? I currently have to go to pypi and then click documentation to check documentation.

**Comment by TAG-Epic** [comment]
You can also press the docs badge! Although I get your point. Will change soontm

**Reply by veni-vidi-code** [reply]
@TAG-Epic reminder

**Comment by TAG-Epic** [comment]
Done!

**Reply by sairam4123** [reply]
thx!
